app/src/full/java/io/homeassistant/companion/android/location/HighAccuracyLocationReceiver.kt

```diff
@@ -17,6 +17,10 @@ class HighAccuracyLocationReceiver : BroadcastReceiver() {
                 {
                     HighAccuracyLocationService.stopService(context)
                     LocationSensorManager.setHighAccuracyModeSetting(context, false)
+                    LocationSensorManager.isBackgroundLocationSetup = false
```

Comment:
Is this required? `LocationSensorManager.ACTION_REQUEST_LOCATION_UPDATES` will go to `setupLocationTracking()` -> `setupBackgroundLocation(true, ?)` -> high accuracy mode is different -> it will try to stop the service and call `requestLocationUpdates()`, which always registers an intent. No override necessary as far as I see.

Reply:
Totally unnecessary, thank you :) 🙏